app/views/password_reset_mailer/reset_notify.text.erb

```diff
@@ -2,7 +2,7 @@ Dear <%= @user.username %>,
 
 Someone (probably you) has requested a reset of your password. To reset your password, click here:
 
-https://<%= ActionMailer::Base.default_url_options[:host] %>/reset/key/<%= @key %>
+http://<%= ActionMailer::Base.default_url_options[:host] %>/reset/key/<%= @key %>
```

Comment:
Hmm, I'm worried this will break the link in production, where we use `https` -- but since we force it, will this resolve to `https` without an error? Thanks for this fix, though!!!

Reply:
@jywarren I added http for development because for https in development it doesn't work.

Reply:
Yup, http will resolve to https in production. We can even try this on ny website online. It resolve http to https.